dotcom-rendering/src/web/experiments/tests/recipe-structured-data.ts

```diff
@@ -0,0 +1,68 @@
+// This is for an experiment testing SEO performance on recipes before
```

Comment:
Could we put this file in a different location, please? This `experiments/` folder is currently only used for AB tests and there is [code which uses these files](https://github.com/guardian/dotcom-rendering/blob/a401bd0fbb839916af4fade22e8d1a3b2974b373/dotcom-rendering/src/web/components/SetABTests.importable.tsx) and expects them to be structured in a certain way.

I would suggest placing it closer to where it is being used and perhaps giving it a very explicit name to indicate that it is to be deleted